apps/frontend/assets/src/omg-page-token-detail/index.js

```diff
@@ -187,12 +186,12 @@ class TokenDetailPage extends Component {
         </DetailGroup>
         <DetailGroup>
           <b>Total Supply:</b>{' '}
-          <span>
+          <span style={{ marginRight: '10px' }}>
             {token.total_supply === undefined
               ? '...'
               : formatReceiveAmountToTotal(token.total_supply, token.subunit_to_unit)}{' '}
             {token.symbol}
-          </span>{' '}
+          </span>
           <Link to={`${this.props.location.pathname}/history`}>view history</Link>
         </DetailGroup>
         <DetailGroup>
```

Comment:
<img width="596" alt="Screen Shot 2019-05-24 at 7 41 19 AM" src="https://user-images.githubusercontent.com/2189889/58295079-7d0e6400-7df7-11e9-86b4-1acccf31b12e.png">
i'm not sure it is from this PR or not, but seems the ui is a bit out